terraform/modules/common/ssh_key/main.tf

```diff
@@ -0,0 +1,18 @@
+## Variables
+
+variable "stack_name" {
+  description = "The name of the stack within which the resource resides"
+}
+
+## Resources
+
+resource "aws_key_pair" "ssh_key" {
+  key_name   = "${var.stack_name}-jumpbox-key"
```

Comment:
Do we want to have `jumpbox` in here as I believe it is supposed to be generic? Potentially instead of taking `stack_name` as a variable to the module we could take `key_name` and let the caller be responsible for naming it?

Reply:
This is also only used by the jumpbox so could move to that project